sklearn/linear_model/_coordinate_descent.py

```diff
@@ -754,22 +754,28 @@ def enet_path(
 
 
 class ElasticNet(MultiOutputMixin, RegressorMixin, LinearModel):
-    """Linear regression with combined L1 and L2 priors as regularizer.
+    r"""Linear regression with combined L1 and L2 priors as regularizer.
 
     Minimizes the objective function::
```

Comment:
The double column indicates a code block. This is what prevent the desired rendering I think.
```suggestion
    Minimizes the objective function:
```